d2l-icon-button.html

```diff
@@ -15,11 +15,12 @@
 			is: 'd2l-icon-button',
 			extends: 'button',
 
-			behaviors: [D2L.PolymerBehaviors.IconButtonBehavior],
+			behaviors: [window.D2L.PolymerBehaviors.IconButtonBehavior],
```

Comment:
@Surekh : something to be aware of - in order for polylint to resolve behavior dependencies, this needs to match the behavior assignment in `d2l-icon-button-behavior.html`.